ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -30,6 +30,12 @@ const CQLUtils = require('../../js/CQLUtils.js')
 const properties = require('../../js/properties.js')
 const Common = require('../../js/Common.js')
 
+const geometryComparators = ['INTERSECTS', 'EMPTY']
+const dateComparators = ['BEFORE', 'AFTER', 'RELATIVE', 'BETWEEN', 'EMPTY']
+const stringComparators = ['CONTAINS', 'MATCHCASE', '=', 'NEAR', 'EMPTY']
+const numberComparators = ['>', '<', '=', '>=', '<=', 'EMPTY']
+const booleanComparators = ['=', 'EMPTY']
```

Comment:
These constant arrays are duplicated in `filter-comparator.view.js`. Is it a common practice to extract these out into a common file and have both components require it?

Reply:
My suspicion here is that these objects should go in `CustomElements.js`. However, I am willing to listen to other ideas

Reply:
There's the `component/filter/filter.js` that right now only exports the default filter. I'm not sure what the best practice is here, though